docs/installation.md

```diff
@@ -39,15 +39,20 @@ two examples.
 Bind-mount your `prometheus.yml` from the host by running:
 
 ```bash
-docker run -p 9090:9090 -v /tmp/prometheus.yml:/etc/prometheus/prometheus.yml \
-       prom/prometheus
+docker run \
+    -p 9090:9090 \
+    -v /tmp/prometheus.yml:/etc/prometheus/prometheus.yml \
+    prom/prometheus
 ```
 
 Or use an additional volume for the config:
 
 ```bash
-docker run -p 9090:9090 -v /prometheus-data \
-       prom/prometheus --config.file=/prometheus-data/prometheus.yml
+docker run \
+    -p 9090:9090 \
+    -v /path/to/data:/prometheus-data \
```

Comment:
I suppose the idea was to show how to [start the container with a volume](https://docs.docker.com/storage/volumes/#start-a-container-with-a-volume). Using a volume name instead of a directory path would be less confusing:

```
docker run \
  -v prometheus:/prometheus-data
  ...
```

cc @SuperQ

Reply:
No, I prefer to show the local path mapping.

I also think we should remove the `--config` option from the example, and show how to pass the config into the default location (`/etc/prometheus/prometheus.ym`) with no additional flags.

By including the one flag override, it breaks the other default flags current `CMD` config.